Resources/Prototypes/Catalog/uplink_catalog.yml

```diff
@@ -175,6 +175,25 @@
   icon: Objects/Misc/id_cards.rsi/default.png
   price: 3
 
+- type: uplinkListing
+  id: UplinkJetpack
+  category: Utility
+  itemId: RevolverCapGun
```

Comment:
This itemId should be something related to the item itself.
Perhaps you forgot to change it?

Reply:
> This itemId should be something related to the item itself. Perhaps you forgot to change it?

I totally missed this, fixed!